src/hiposfer/kamal/services/routing/core.clj

```diff
@@ -90,8 +90,9 @@
     (data/transact! conn (cache-stop-successors @conn))
     (conj ag conn)))
 
-(defn- pedestrian-graph
-  "builds a datascript in-memory db and conj's it into the passed agent"
+(defn pedestrian-graph
+  "builds a datascript in-memory db and returns it. Only valid for
```

Comment:
I read about datascript and datalog. It is new to me, so I just used this chance to learn more about it.

Reply:
@mehdisadeghi I recommend you these websites:
- http://www.learndatalogtoday.org/
- http://udayv.com/clojurescript/clojure/2016/04/28/datascript101/

I used them to understand the concepts :)